com.hazelcast.jet.impl.connector.JmsSourceIntegration_NonSharedClusterTest.when_memberTerminated_then_transactionsRolledBack

_master_ (commit 55958e30a94d419f740be754876f94fc8dc58f1f)

Failed on IBM JDK8: http://jenkins.hazelcast.com/job/jet-oss-master-ibm-jdk8/249/testReport/junit/com.hazelcast.jet.impl.connector/JmsSourceIntegration_NonSharedClusterTest/when_memberTerminated_then_transactionsRolledBack/

Stacktrace:
```
java.lang.AssertionError: expected items not in sink expected:<10000> but was:<9207>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at com.hazelcast.jet.impl.connector.JmsSourceIntegration_NonSharedClusterTest.lambda$when_memberTerminated_then_transactionsRolledBack$0(JmsSourceIntegration_NonSharedClusterTest.java:84)
	at com.hazelcast.jet.impl.connector.JmsSourceIntegration_NonSharedClusterTest$$Lambda$1545.000000003C684D90.run(Unknown Source)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1356)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1373)
	at com.hazelcast.jet.impl.connector.JmsSourceIntegration_NonSharedClusterTest.when_memberTerminated_then_transactionsRolledBack(JmsSourceIntegration_NonSharedClusterTest.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.hazelcast.test.DumpBuildInfoOnFailureRule$1.evaluate(DumpBuildInfoOnFailureRule.java:37)
	at com.hazelcast.test.jitter.JitterRule$1.evaluate(JitterRule.java:104)
	at com.hazelcast.test.metrics.MetricsRule$1.evaluate(MetricsRule.java:62)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:277)
	at java.lang.Thread.run(Thread.java:818)
```

Standard output:
```
Hiccups measured while running test 'when_memberTerminated_then_transactionsRolledBack(com.hazelcast.jet.impl.connector.JmsSourceIntegration_NonSharedClusterTest):'
01:57:50, accumulated pauses: 435 ms, max pause: 87 ms, pauses over 1000 ms: 0
01:57:55, accumulated pauses: 344 ms, max pause: 21 ms, pauses over 1000 ms: 0
01:58:00, accumulated pauses: 625 ms, max pause: 178 ms, pauses over 1000 ms: 0
01:58:05, accumulated pauses: 533 ms, max pause: 216 ms, pauses over 1000 ms: 0
01:58:10, accumulated pauses: 601 ms, max pause: 145 ms, pauses over 1000 ms: 0
```

--- a/hazelcast-jet-core/src/test/java/com/hazelcast/jet/impl/connector/JmsSourceIntegration_NonSharedClusterTest.java
+++ b/hazelcast-jet-core/src/test/java/com/hazelcast/jet/impl/connector/JmsSourceIntegration_NonSharedClusterTest.java
@@ -81,7 +81,7 @@ public class JmsSourceIntegration_NonSharedClusterTest extends JetTestSupport {
                 .setProcessingGuarantee(EXACTLY_ONCE)
                 .setSnapshotIntervalMillis(DAYS.toMillis(1)));
 
-        assertTrueEventually(() -> assertEquals("expected items not in sink", MESSAGE_COUNT, sinkList.size()), 5);
+        assertTrueEventually(() -> assertEquals("expected items not in sink", MESSAGE_COUNT, sinkList.size()), 20);
 
         // Now forcefully shut down the second member. The terminated member
         // will NOT roll back its transaction. We'll assert that the
@@ -91,7 +91,7 @@ public class JmsSourceIntegration_NonSharedClusterTest extends JetTestSupport {
         // transaction will be stalled and only emitted once, they will be
         // emitted after the default Artemis timeout of 5 minutes.
         instance2.getHazelcastInstance().getLifecycleService().terminate();
-        assertTrueEventually(() -> assertEquals("items should be emitted twice", MESSAGE_COUNT * 2, sinkList.size()), 20);
+        assertTrueEventually(() -> assertEquals("items should be emitted twice", MESSAGE_COUNT * 2, sinkList.size()), 30);
     }
 
     @Test

